app/src/main/java/si/inova/zimskasola/ActivityMain.kt

```diff
@@ -324,7 +325,7 @@ class ActivityMain: AppCompatActivity(), MrBeacon.Listener, OnLocationUpdatedLis
     private fun startLocation() {
         // testing start
         // ustvari ograjo okol moje tesne lokacije, drugače je treba ta naslov prebrat iz jsona (naslov stavbe)
```

Comment:
You should write comments in English language, so the developers after you can understand them :)